rna_design/rna_synthesizer_n.py

```diff
@@ -0,0 +1,316 @@
+#!/usr/bin/env python
```

Comment:
lets rename the folder rna_design as rnasynth
since it is the name of the package